.github/workflows/sbom-attest.yml

```
@@ -0,0 +1,99 @@
+      packages: write # Used to push images to `ghcr.io` if used.
+      id-token: write # Needed to create an OIDC token for keyless signing
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Install Cosign
+        uses: sigstore/cosign-installer@faadad0cce49287aee09b3a48701e75088a2c6ad  # v4.0.0
+      - name: Authenticate to Registry
+        uses: docker/login-action@5e57cd118135c172c3672efd75eb46360885c0ef  # v3.6.0
+        with:
+          registry: ${{ inputs.registry }}
+          username: ${{ inputs.registry_username }}
+          password: ${{ inputs.registry_password }}
```

actionlint (1.7.12)

[error] property "registry_password" is not defined in object type {image_digest: string; image_name: string; registry: string; registry_username: string; sbom-predicate-type: string; sbom-repository: string}

(expression)